python/ql/consistency-queries/qlpack.yml

```diff
@@ -0,0 +1,5 @@
+name: codeql/python-consistency-queries
+groups: [python, consistency-queries]
```

Comment:
Should we add this to the other packs as well?

Reply:
Should we also add the `test` group? If I'm not mistaken QL packs in the `test` group are not published.  @aeisenberg is that right?

Reply:
Good point :+1: Truth be told, I made the commits for streamlining the other languages and the one for adding the Python consistency queries on two separate days (originally in 2 separate branches), so I had forgotten about this detail.

Reply:
Yes. Since they are now part of the workspace, they will be picked up when publishing.

Note that if you remove the `groups` property, the they will be ignored. So, either add the `test` group here and to all consistency query packs. Or remove this group.

I'd prefer adding the test group here and else where since we may want to specify this in the future